ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-gas-limit/advanced-gas-fee-gas-limit.js

```diff
@@ -66,7 +66,9 @@ const AdvancedGasFeeGasLimit = () => {
       tag={TYPOGRAPHY.Paragraph}
       variant={TYPOGRAPHY.H7}
       className="advanced-gas-fee-gas-limit"
-      margin={[4, 2, 0, 2]}
+      marginTop={4}
+      marginLeft={2}
+      marginRight={2}
     >
       <strong>
         <I18nValue messageKey="gasLimitV2" />
```

Comment:
Updating previous array props for different directions to singular direction props now that array props are reserved for responsive props